examples/inverse/plot_mne_crosstalk_function.py

```diff
@@ -55,15 +55,16 @@
     inverse_operator, forward, labels, method=method, lambda2=lambda2,
     signed=False, mode=mode, n_svd_comp=n_svd_comp)
 
-time_label = "MNE %d"
-brain_mne = stc_ctf_mne.plot(hemi='rh', subjects_dir=subjects_dir,
-                             time_label=time_label,
-                             figure=create_3d_figure(size=(500, 500)))
+with use_3d_backend("mayavi"):
+    time_label = "MNE %d"
+    brain_mne = stc_ctf_mne.plot(hemi='rh', subjects_dir=subjects_dir,
+                                 time_label=time_label,
+                                 figure=create_3d_figure(size=(500, 500)))
```

Comment:
You can't use `figure=1` here and `figure=2` below?

Reply:
I just missed this example, thank you